controllers/helpers/ctrlWrapper.js

```diff
@@ -0,0 +1,15 @@
+const ctrlWrapper = (ctrl) => {
```

Comment:
Why do you need this? :)

Reply:
I tried to find a solution for centrally handling errors that may occur in controllers, but I couldn't and wrote a path for every controller. But I saved this module for the future :)